website/scripts/rosbridge.js

```diff
@@ -65,6 +65,7 @@ var t_nav_sat_fix = '';
 var t_robot_status = '';
 var t_cmd_speed = '';
 var t_lateral_control_driver = '';
+var t_light_bar_status = '/control/light_bar_status'; //02/2019: added to display lightbar on UI
```

Comment:
should be base topic name "control/light_bar_status" without the starting "/"

Reply:
Tested on the vehicles and local environment, it works with the slash, "/control/light_bar_status".

Reply:
What was the remapping you used?
Having a "/" in the front makes this a global namespace meaning out wrapper namespace like saxton_cav/control/light_bar_status might not be applied. All the other topics in this file are missing the "/"